modules/exploits/example_webapp.rb

```diff
@@ -0,0 +1,145 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+###
+#
+# This exploit sample shows how an exploit module could be written to exploit
+# a bug in an arbitrary web server
+#
+###
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = NormalRanking
+
+  #
+  # This exploit affects a webapp, so we need to import HTTP Client
+  # to easily interact with it.
+  #
+  include Msf::Exploit::Remote::HttpClient
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        # The Name should be just like the line of a Git commit - software name,
+        # vuln type, class. It needs to fit in 50 chars ideally. Preferably apply
+        # some search optimization so people can actually find the module.
+        # We encourage consistency between module name and file name.
+        'Name'           => 'Sample Webapp Exploit',
+        'Description'    => %q(
+            This exploit module illustrates how a vulnerability could be exploited
+          in a webapp.
+        ),
+        'License'        => MSF_LICENSE,
+        'Author'         => ['h00die'],
+        'References'     =>
+          [
+            [ 'OSVDB', '12345' ],
+            [ 'EDB', '12345' ],
+            [ 'URL', 'http://www.example.com'],
+            [ 'CVE', '1978-1234']
+          ],
+        'Platform'       => ['python'],
+        'Privileged'     => false,
+        'Arch'           => ARCH_PYTHON,
+        'Targets'        =>
+          [
+            [ 'Automatic Target', {}]
+          ],
+        'DisclosureDate' => "Apr 1 2013",
+        # Note that this is by index, rather than name. It's generally easiest
```

Comment:
```suggestion
        # Note that DefaultTarget refers to the index of an item in Targets, rather than name. It's generally easiest
```
I get it's wordier but explicitly stating the relationship like something above would make this much simpler

Reply:
i like it, this was a copy from example.rb, ill update it there as well.